features/steps/unity_steps.rb

```diff
@@ -0,0 +1,17 @@
+When("I build a Unity application") do
+  run_required_commands([
+    ["features/scripts/create_unity_project.sh"]
+  ])
+end
+
+When("run the application") do
+  run_required_commands([
+    ["features/scripts/launch_unity_application.sh"]
+  ])
+end
+
+Given("I configure the bugsnag endpoint") do
```

Comment:
May want to consider renaming this to 'bugsnag notify endpoint' for when we add sessions